testsuite/test-modprobe.c

```diff
@@ -244,9 +244,7 @@ DEFINE_TEST(modprobe_param_kcmdline_show_deps,
 	},
 	.output = {
 		.out = TESTSUITE_ROOTFS "test-modprobe/module-param-kcmdline/correct.txt",
-	},
-	.modules_loaded = "",
-	);
+	});
```

Comment:
As-is this test also ensures that we are not unintentionally loading the module, as part of `--show-depends`. I suspect it's a case we want to check for plus equivalent ones for the other `--show*` options.

The core purpose of test is to check for something else, so :+1: from me.

Let me know if you see value in the test cases mentioned above and we'll add them to the todo list.